api/app/signals/apps/signals/migrations/0129_new_subcategory_kerstbomen.py

```diff
@@ -0,0 +1,66 @@
+"""
+Category changes SIG-3347
+
+New subcategory for Kerstbomen (christmas trees)
+"""
+from django.db import migrations
+
+# SIG-3347
+NEW_CATEGORIES = {
+    'afval': {
+        'kerstbomen': {
+            'name': 'Kerstbomen',
+            'description': 'Meldingen over aangeboden/gedumpte Kerstbomen',
+            'handling': 'A3DMC',
```

Comment:
We should revisit these `handling` codes, in the past they were a key to look up the `handling_message`. I see that they no longer uniquely identify a specific `handling_message` (as expected given the db schema).

Reply:
Agreed, not for this PR but we can remove this column because it is no longer used